pkg/builder/builder_test.go

```diff
@@ -230,8 +231,29 @@ var _ = Describe("builder", func() {
 		Expect(c.SecurityContext.Sysctls[1].Name).Should(Equal("net.ipv4.tcp_keepalive_intvl"))
 		Expect(c.SecurityContext.Sysctls[1].Value).Should(Equal("5"))
 	})
+
+	It("should mount ssh secret for dbadmin and root", func() {
+		vdb := vapi.MakeVDB()
+		vdb.Spec.SSHSecret = "my-secret"
+		c := buildPodSpec(vdb, &vdb.Spec.Subclusters[0], &DeploymentNames{})
+		cnt := &c.Containers[0]
+		i, ok := getFirstSSHSecretVolumeMountIndex(cnt)
+		Expect(ok).Should(BeTrue())
+		Expect(cnt.VolumeMounts[i].MountPath).Should(Equal(paths.DBAdminSSHPath))
```

Comment:
I think this will always fail

Reply:
I made my last change too fast.

Reply:
Sorry if the comment was confusing but I saw that you fixed the tests.